auth-server/src/schema/mutation/changeNotificationPreference.ts

```diff
@@ -0,0 +1,8 @@
+export default `
+    changeNotificationPreference(
+        post_participated: Boolean
+        post_created: Boolean
+        new_proposal: Boolean
+        own_proposal: Boolean
```

Comment:
we'd use this `notificationPreferencesType` here as well.

Reply:
done